src/Core/src/Fonts/EmbeddedFontLoader.Android.cs

```diff
@@ -1,8 +1,22 @@
 #nullable enable
 
+using System;
+using System.IO;
+
 namespace Microsoft.Maui
 {
 	public partial class EmbeddedFontLoader : FileSystemEmbeddedFontLoader
 	{
+		public EmbeddedFontLoader(IServiceProvider? serviceProvider = null)
+			: base(GetTempPath, serviceProvider)
```

Comment:
The difference between the Android API and `System.IO.Path.GetTempPath()` appears to be a trailing slash:
```
03-07 14:18:36.776 19754 19754 I DOTNET  : Path: /data/user/0/com.companyname.foo/cache/
03-07 14:18:36.787 19754 19754 I DOTNET  : Android: /data/user/0/com.companyname.foo/cache
```
Maybe let's just change this to this?
```suggestion
			: base(Path.GetTempPath, serviceProvider)
```
And delete `GetTempPath()`?

Reply:
I'm not sure I feel comfortable changing the behavior here. Are we certain that your suggestion is safe?

Reply:
If they are looking for a temp folder to extract fonts to, it seems like `Path.GetTempPath()` is what they would want? I don't think you need to call into Java to get this path.

I think they would only use this folder if a font is an `EmbeddedResource`, because there isn't an Android API for loading a font from a stream.

Reply:
Looks like this code was added in https://github.com/dotnet/maui/pull/1352, which says:

> Avoid the use of Path.GetTempPath() in the font loader because the first access to IO is slow. move it on a bit and see what changes

Reply:
cc @mattleibow

Reply:
I got this info from @grendel and we changed it because it was expensive.

Reply:
So we no longer use the filesystem in the font startup path anymore. So if this was just added for perf reasons, can I change it back to just `Path.GetTempPath`?

Reply:
Yeah, that makes sense. Keep it nice and clean.